docs/portfolio.md

```diff
@@ -45,6 +45,8 @@ We are especially interested in projects supporting researchers, focused on rese
   
 **[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
 
+**[Rerum Admin Work](project_rerum_admin_work/about.md)** - A tiny NodeJS web application for connecting with the RERUM API. This is for software developers looking to develop a client application that uses the public RERUM API as the client's back end.
```

Comment:
Please move this to the External Projects section that @MutyaSowmya123 created. It's merged into main, so you can either merge main into your branch or rebase your branch off of main to get this section.